scripts/gulp/clean.js

```diff
@@ -1,10 +1,10 @@
 'use strict';
 
-var gulp = require( 'gulp' );
-var plugins = require( 'gulp-load-plugins' )();
-var component = require('./parseComponentName');
+const component = require('./parseComponentName');
```

Comment:
Not required here since this file isn't consistent, but at some point I'd like to see us bring these gulp files up to formatting standard.